apps/alchimie/alchimie/definitions.py

```
@@ -1,10 +1,27 @@
-from dagster import Definitions, load_assets_from_modules
+from dagster import Definitions, load_assets_from_modules, ResourceDefinition
+import psycopg2
 
 from alchimie import assets  # noqa: TID252
 from alchimie.jobs import my_k8s_job
+
 all_assets = load_assets_from_modules([assets])
 
+
+def get_sql_conn():
```

Ruff (0.8.2)

Missing return type annotation for public function `get_sql_conn`

(ANN201)